data-import/desktop-csv-import/desktop-csv-import.adoc

```diff
@@ -0,0 +1,420 @@
+
+There was one tricky thing with this CSV in the `orderDate` column.
+Neo4j's datetime format will accept many formats, but the time value must start with a `T`.
+The CSV file did not have the 'T' joining the date and time values but had a space between them instead.
+We used the `replace()` function to change the space to the character 'T' and get the string into the expected format.
+Then, we wrapped the `datetime()` function around that to convert the changed string to a datetime value.
+
+.Order-details.csv
+The values in the order-details.csv (from column names) are for productID, orderID, and quantity.
+Let us look at which ones will need to be converted.
+
+Our product ID is also from our products.csv file, where we converted that value to an integer.
+We will do the same here to ensure we match formats.
+The order ID field contains values from our orders.csv file, so we will match our previous conversion and translate this field to an integer, as well.
+The quantity field in this file is a numeric value.
+We can convert this to an integer with the `toInteger()` function we have been using.
+
+The results of these conversions are in the code below.
+Remember that we still are not loading any data yet.
+
+[source, cypher]
+----
+LOAD CSV WITH HEADERS FROM 'file:///desktop-csv-import/order-details.csv' AS row
+RETURN toInteger(row.productID), toInteger(row.orderID), toInteger(row.quantity)
+LIMIT 8
+----
+
+image::{img}desktop_import_details_convert.jpg[link="{img}desktop_import_details_convert.jpg",role="popup-link"]
+
+== Loading the data!
+
+Now that we have determined that the CSV file data looks ok, and we have verified how `LOAD CSV` sees the data and converted any non-string values, we are almost ready to create the data in our graph database!
+To do that, we use Cypher statements alongside the `LOAD CSV` commands we used above.
+The `LOAD CSV` will read the files, and the Cypher statements will create the data in our database.
+
+=== Graph data model
+
+An important step we need before writing Cypher statements, though, is to determine what the graph structure should look like once we import our file data.
+After all, importing the data in the existing table and column data will not provide the value we want to achieve from a graph.
+To utilize the graph database fully, we need a graph data model.
+
+We do not need to analyze all the various ways we could organize the products and orders in our files, so we have provided the model we will use for this exercise below.
+
+image::{img}desktop_import_data_model.jpg[link="{img}desktop_import_data_model.jpg",role="popup-link"]
+
+We have two nodes - one for a product and one for an order.
+Each of those nodes have properties that we have from our CSV files.
+For the `Product`, we have ID, name, and unit cost.
+For the `Order`, we have ID, date/time, and country where it is going.
```

Comment:
we _could_ turn country into a node - why/why-not

Reply:
That's a debate that would take time to explain fully, which might be better suited for a data modeling guide. I definitely agree that this model is very narrow in scope, but I think it's required to focus solely on import and leave other topics for other guides. Do you think that's a reasonable approach?